engine/classes/Elgg/Database/ConfigTable.php

```diff
@@ -96,24 +96,6 @@ function set($name, $value) {
 			':name' => $name,
 			':value' => serialize($value),
 		];
-		
```

Comment:
Sync upgrades will always run after database migrations, so this is not needed anymore